### ISSUE TITLE
Would be nice to have a flag for bots

### ISSUE THREAD

**Issue by hackhat** [antivanov/sniffr#5]

**Comment by antivanov** [comment]
Yes, this can be a nice feature, I will investigate a bit whether we can determine easily from the user agent string if the agent is a bot.

**Comment by hackhat** [comment]
I've researched a bit, the cheap option would be to check for the string "bot" case insensitive. The other option is to get the list of bot useragents from somewhere...
